lock-impl/src/main/java/com/palantir/lock/impl/LockServiceImpl.java

```diff
@@ -471,20 +469,24 @@ private void tryLocks(LockClient client, LockRequest request, BlockingMode block
         }
     }
 
-    private void logSlowLockAcquisition(String lockId, LockClient currentHolder, long duration) {
-        if (slowLogEnabled) {
+    private void logSlowLockAcquisition(String lockId, LockClient currentHolder, long durationMillis) {
+        if (isSlowLogEnabled() && durationMillis >= slowLogTriggerMillis) {
             SlowLockLogger.logger.info("Blocked for {} ms to acquire lock {} {}.",
-                    duration,
+                    durationMillis,
                     lockId,
                     currentHolder == null ? "successfully" : "unsuccessfully");
-        } else {
+        } else if (durationMillis > 100) {
```

Comment:
This was in the interest of preserving the bebug logging functionality, see line 454.